test/functional/feature_config_args.py

```diff
@@ -135,14 +134,15 @@ def test_args_log(self):
                 ],
                 unexpected_msgs=[
                     'alice:f7efda5c189b999524f151318c0c86$d5b51b3beffbc0',
-                    '127.1.1.1',
                     'secret-rpcuser',
                     'secret-torpassword',
                 ]):
             self.start_node(0, extra_args=[
                 '-addnode=some.node',
                 '-rpcauth=alice:f7efda5c189b999524f151318c0c86$d5b51b3beffbc0',
                 '-rpcbind=127.1.1.1',
+                '-rpcbind=127.0.0.1',
+                "-rpcallowip=127.0.0.1",
```

Comment:
Are those additions left-overs from local testing? I think keeping `-rpcbind`/`-rpcallowip` would only make sense if they would somehow appear in `expected_msgs/unexpected_msgs` (could add e.g. `rpcbind=****` on `unexpected_msgs` to check that it is _not_ sensitive anymore), otherwise the test coverage is not increased in any way.

Reply:
Added them to `unexpected_msgs`.